src/DependencyInjection/SimpleThingsEntityAuditExtension.php

```diff
@@ -49,5 +49,26 @@ public function load(array $configs, ContainerBuilder $container): void
                 $container->setAlias('simplethings_entityaudit.'.$key, $service);
             }
         }
+
+        $this->fixParametersFromDoctrineEventSubscriberTag($container, [
+            'simplethings_entityaudit.log_revisions_listener',
+            'simplethings_entityaudit.create_schema_listener',
+        ]);
+    }
+
+    private function fixParametersFromDoctrineEventSubscriberTag(ContainerBuilder $container, array $definitionNames): void
+    {
+        foreach ($definitionNames as $definitionName) {
+            $definition = $container->getDefinition($definitionName);
+            $tags = $definition->getTag('doctrine.event_subscriber');
+            $definition->clearTag('doctrine.event_subscriber');
+
+            foreach ($tags as $attributes) {
+                if (isset($attributes['connection'])) {
+                    $attributes['connection'] = (string) $container->getParameter('simplethings.entityaudit.connection');
```

Comment:
IMO, if we harcode the parameter name here, we are making less flexible the replacing mechanism.
If in the future the parameter name used in these tags is renamed, we must replace this occurrence too.
I also think that the final users could not override these services if they need to use a different value.

Reply:
We set this `simplethings.entityaudit.connection` parameter in the same `SimpleThingsEntiyAuditExtension`, so I think we do not have to afriaid about rename. Additionaly if users override this service then connection name must be set directly, oherwise expection will be throw. 

We can also move it to the compiler pass. Then we can check whole services tagged by `doctrine.event_listener` and replace `simplethings.entityaudit.connection` parameter in `connetion` to propertly value.

Reply:
https://github.com/symfony/symfony/pull/40262
WDYT? @sonata-project/contributors

Reply:
I'm not sure to understand, should we wait for https://github.com/symfony/symfony/pull/40262 or can we merge this ?

Reply:
We can merge this now. This code allow use param only for ower services.we can remove it when symfony accept it and we bump doctrine-bridge.